addons/packages/package-overlay/package-overlay.yaml

```diff
@@ -1,6 +1,12 @@
 #@ load("@ytt:overlay", "overlay")
 #@ load("@ytt:data", "data")
 #@ load("@ytt:yaml", "yaml")
+#@overlay/match by=overlay.subset({"kind":"Package"}),expects="1+"
+---
+spec:
+#@overlay/match missing_ok=True
+  valuesSchema: {}
```

Comment:
Otehrwise, ytt will through error saying no match for `spec.valuesSchema`.

Reply:
I fixed this in kapp controller PR. So you can rebase now

Reply:
@blc1996 , I think this file is not successfully rebased. It has repetitions. I fixed this in my MR, so you shouldn't see any changes for this file

Reply:
Removed